install_scripts/bootstrap_arch_linux.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+# Get shared functions
+source ./common.sh
+
+# Configuration
+SADM_LOCALE='en_US.UTF-8'
+SADM_CHARSET='UTF-8'
+SADM_TIMEZONE='Europe/Paris'
+
+# Arch Linux install
+ARCH_MIRROR=http://mirror.rackspace.com/archlinux
+# Release used for bootstraping from a non-Arch Linux system
+ARCH_RELEASE_DATE=2017.04.01
+
+# Usage
+if [ $# -ne 3 ]; then
+    echo >&2 "Usage: $0 ROOT_DIR HOSTNAME PLAINTEXT_ROOT_PASSWORD_FILE"
+    echo >&2 ""
+    echo >&2 "Install and configure Arch Linux in the ROOT_DIR folder."
+    echo >&2 "Example (as root):"
+    echo >&2 "  $ echo my_root_password_is_pretty_long > plaintext_root_pass"
+    echo >&2 "  $ mkdir gw"
+    echo >&2 "  $ ./bootstrap_arch_linux.sh ./gw gw ./plaintest_root_pass¬"
+    exit 1
+fi
+
+# Requirements checks
+if test -e /etc/arch-release && ! which arch-chroot >/dev/null; then
+    echo >&2 "Error: This script requires arch-install-scripts, please run:"
+    echo >&2 "> pacman -S arch-install-scripts"
+    exit 1
+fi
+
+# This script creates a filesystem as root
+this_script_must_be_run_as_root
+
+# Get command line args
+if [ ! -d $1 ]; then
+    echo >&2 "Error: '$root_dir': no such directory"
+    exit 1
+fi
+
+# Argument parsing
+root_dir="$(readlink --canonicalize $1)"
+hostname="$2"
+root_password_file="$3"
+
+if [ ! -r $root_password_file ]; then
+    echo >&2 "Error: password file '$root_password_file' must be readable"
+    exit 1
+fi
+
+# The actual Arch Linux setup starts here
+echo "[+] Installing base Arch Linux"
```

Comment:
This looks like you wanted to use `echo_status` for all the `[+] foo` lines, don't you?